docs/examples/bifacial/plot_irradiance_nonuniformity_loss.py

```diff
@@ -0,0 +1,125 @@
+# the Relative Mean Absolute Difference (RMAD) of the irradiance of each cell.
+# They did also use the standard deviation of the cells' irradiances, but they
+# found that the RMAD was a better predictor of the mismatch loss.
+#
+# This example demonstrates how to model the irradiance non-uniformity loss
+# from the irradiance levels of each cell in a PV module.
+#
+# The function
+# :py:func:`pvlib.bifacial.power_mismatch_deline` is
+# used to transform the Relative Mean Absolute Difference (RMAD) of the
+# irradiance into a power loss percentage. Down below you will find a
+# numpy-based implementation of the RMAD function.
+#
+# References
+# ----------
+# .. [1] C. Deline, S. Ayala Pelaez, S. MacAlpine, and C. Olalla, 'Estimating
+#    and parameterizing mismatch power loss in bifacial photovoltaic
+#    systems', Progress in Photovoltaics: Research and Applications, vol. 28,
+#    no. 7, pp. 691-703, 2020, :doi:`10.1002/pip.3259`.
+#
+# .. sectionauthor:: Echedey Luis <echelual (at) gmail.com>
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.cm import ScalarMappable
+from matplotlib.colors import Normalize
+
+from pvlib.bifacial import power_mismatch_deline
+
+# %%
+# Problem description
+# -------------------
+# Let's set a fixed irradiance to each cell row of the PV array with the values
+# described in Figure 1 (A), [1]_. We will cover this case for educational
+# purposes, although it can be achieved with the packages
+# :ref:`solarfactors <https://github.com/pvlib/solarfactors/>` and
+# :ref:`bifacial_radiance <https://github.com/NREL/bifacial_radiance>`.
+#
+# Here we set and plot the global irradiance level of each cell.
+
+x = np.arange(12, 0, -1)
+y = np.arange(6, 0, -1)
+cells_irrad = np.repeat([1059, 976, 967, 986, 1034, 1128], len(x)).reshape(
+    len(y), len(x)
+)
+
+color_map = "gray"
+color_norm = Normalize(930, 1150)
+
+fig, ax = plt.subplots()
+fig.suptitle("Global Irradiance Levels of Each Cell")
+fig.colorbar(
+    ScalarMappable(cmap=color_map, norm=color_norm),
+    ax=ax,
+    orientation="vertical",
+    label="$[W/m^2]$",
+)
+ax.set_aspect("equal")
+ax.pcolormesh(
+    x,
+    y,
+    cells_irrad,
+    shading="nearest",
+    edgecolors="black",
+    cmap=color_map,
+    norm=color_norm,
+)
+
+# %%
+# Relative Mean Absolute Difference
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Calculate the Relative Mean Absolute Difference (RMAD) of the cells'
+# irradiances with the following function, Eq. (4) of [1]_:
+#
+# .. math::
+#
+#    \Delta \left[ \% \right] = \frac{1}{n^2 \bar{G}_{total}}
+#    \sum_{i=1}^{n} \sum_{j=1}^{n} \lvert G_{total,i} - G_{total,j} \rvert
+#
+
+
+def rmad(data, axis=None):
+    """
+    Relative Mean Absolute Difference. Output in percentage.
+    https://stackoverflow.com/a/19472336/19371110
+    """
+    mad = np.mean(np.absolute(data - np.mean(data, axis)), axis)
+    return mad / np.mean(data, axis) * 100
```

Comment:
Oops, I think I gave a bad suggestion.  If we are writing the function to have unitless inputs and outputs, we should probably stick with unitless calculations here in the example as well.  Sorry!